espresso-support/demo/src/androidTest/java/com/novoda/movies/MovieItemViewTalkBackTest.java

```diff
@@ -57,6 +61,18 @@ public void clickMovieItemView() {
         checkMenuDisplayed();
     }
 
+    @Test
+    public void movieItemViewHasCustomUsageHintOnClick() {
+        onView(withClassName(is(MovieItemView.class.getName())))
+                .check(matches(withUsageHintOnClick("see actions")));
+    }
+
+    @Test
+    public void movieItemViewHasCustomUsageHintOnLongClick() {
+        onView(withClassName(is(MovieItemView.class.getName())))
+                .check(matches(withUsageHintOnLongClick("open details")));
```

Comment:
TalkBack is the only one I know that uses usage hints but it's not the only service that has access to them. All accessibility services do ✨ 

For me, I'd rather keep this - not for future compatibility, but making it case-_in_sensitive would be weakening the default, with no compelling reason to do so. Unless you have one?

usually this would be a resource string, but I think I was lazy and used the same usage hint as the action label, then made it lowercase in code.